src/language-markdown/pragma.js

```diff
@@ -9,6 +9,7 @@ function startWithPragma(text) {
   const regex = new RegExp(
     [
       `<!--\\s*${pragma}\\s*-->`,
+      `{\\s*\\/\\*\\s*${pragma}\\s*\\*\\/\\s*}`,
       `<!--.*\r?\n[\\s\\S]*(^|\n)[^\\S\n]*${pragma}[^\\S\n]*($|\n)[\\s\\S]*\n.*-->`,
```

Comment:
This other regex looks complex, which is why I didn’t port that to an es-comment either